bricks/multipart_request_handler/README.md

```diff
@@ -0,0 +1,33 @@
+# local_auth
+
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+
+A brick to add multipart_request_handler service to your project
```

Comment:
Please describe your usecase, why was dio not enough so that people know when they need this little brick of yours

Reply:
Added description in the README